### PR TITLE
Framework

### PR DESCRIPTION
**Pull request by a81884855**
Pass the Circle CI test,

### CODE REVIEW

File: client/src/WelcomeMessage.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    marginBottom: 15,
+  },
+  logo: {
+    width: 250,
+    heading: 250,
+    objectFit: 'cover',
+  },
+};
+
+const WelcomeMessage = ({ imgPath }) => {
```
Comment (M-Shehu) [inline client/src/WelcomeMessage.jsx]: @a81884855 What is WeclomeMessage.jsx and why do you need it?

File: client/src/WelcomeMessages.test.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { shallow } from 'enzyme';
+
+// Components
+import WelcomeMessage from './WelcomeMessage.jsx';
```
Comment (M-Shehu) [inline client/src/WelcomeMessages.test.jsx]: @a81884855 I assume this is the default test to check if Enzyme is working. Kindly confirm

File: client/src/averagePrice.jsx [hunk 3]
```diff
@@ -1,101 +1,118 @@
...
+        }
+      }
+      for (let i = 0; i < allPriceBar.length; i += 1) {
+        allData[i] = [];
+        allData[i].push(collectPriceData[i][0]);
+        allData[i].push(collectPriceData[i][1]);
+        if (allPriceBar[i] === undefined) {
+          allData[i].push(1);
+        } else {
+          allData[i].push(allPriceBar[i].length);
```
Comment (M-Shehu) [inline client/src/averagePrice.jsx]: Its good practice to abstract business logic out of the render function of a function. I would advice you refactor to have the business logic in a different class and call that class within AveragePrice

File: client/styles.css [hunk 4]
```diff
@@ -3,3 +3,65 @@ body {
  color: black;
 }
 
+p#rightNow {
+ font-size: 13px;
```
Comment (M-Shehu) [inline client/styles.css]: Good job on the css. For the purposes of the proxy server setup, kindly refactor your class and ID names to include the name of your component

File: package.json [hunk 5]
```diff
@@ -9,30 +9,39 @@
...
+    "cors": "^2.8.5",
+    "express": "^4.16.4",
+    "faker": "^4.1.0",
+    "jquery": "^3.3.1",
+    "mongoose": "^5.4.20",
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
   },
   "devDependencies": {
-    "babel-core": "^6.23.1",
```
Comment (M-Shehu) [inline package.json]: Good job updating dependencies

File: client/src/WelcomeMessage.jsx (hunk 1 above)
Comment (a81884855) [reply client/src/WelcomeMessage.jsx]: This one is from Enzyme test

**Pull request merged**